src/couch/src/couch_httpd.erl

```diff
@@ -110,7 +110,7 @@ start_link(Name, Options) ->
     {ok, ServerOptions} = couch_util:parse_term(
         config:get("httpd", "server_options", "[]")),
     {ok, SocketOptions} = couch_util:parse_term(
-        config:get("httpd", "socket_options", "[]")),
+        config:get("httpd", "socket_options", "[{sndbuf, 262144}]")),
```

Comment:
Hmm it seems this bit of code was duplicated with one right above it, we do `{ok, ServerOptions} =` twice in a row. Wonder what happened there. I think we'd want to do it once only and then apply the new defaults.

Reply:
In default.ini, ServerOptions was commented out before, so it should get `[]`;
However, `socket_options = [{sndbuf, 262144}]`, so I changed it here.
Thanks for the review. @nickva

Reply:
Oh, you're right! For some reason I read both lines as `{ok, ServerOptions}`. Sorry about that. Yeah it looks good with a macro default.